all_test.go

```diff
@@ -41,11 +42,11 @@ func TestMain(m *testing.M) {
 
 	generateUser(qdbUserAdd)
 	generateClusterKeys(qdbClusterKeygen)
-	unsecuredDB, err = newDB(qdbd, SecurityNone)
+	unsecuredDB, err = newDB(qdbd, SecurityNone, false)
```

Comment:
Hmm, could you use a different type than `bool`? Something analogical to `SecurityNone` will be perfect. `ClusterPurgeEnabled`?